system-tests/services/_scripts/setup-external-volumes

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
@tnguyen0 Let's please not use the `setJSON` helper to provision/set up the cluster for the respective test.